### PR TITLE
Add addition of Operator with Tensor

### LINKED ISSUE #387: Add OperatorTensorSum
Allow for the addition of a tensor with an operator, e.g. useful for Tikonov-regularised SENSE where we need `A^HA + lambda*UnityMatrix`

lambda will be broadcasted to the output of the operator if possible.

### PR DESCRIPTION
**Pull request by ckolbPTB**
closes #387

### CODE REVIEW

**Comment by fzimmermann89** [comment]
Do we want 
Operator + lambda to mean
a) (Operator+ lambda)(x)=Operator(x) + lambda(x) or
b) Operator(x) + lambda

For regularized reconstruction we will need a)

**Comment by fzimmermann89** [comment]
B) would not be a linear operator 

Something I realized after writing the code for it.... ...

**Comment by fzimmermann89** [comment]
And a)ight indeed br better served by a identity operator, see #390, last test.

File: src/mrpro/operators/LinearOperator.py [hunk 1]
```diff
@@ -151,21 +152,32 @@ def __matmul__(
         else:
             return OperatorComposition(self, other)
 
-    @overload
-    def __add__(self, other: LinearOperator) -> LinearOperator: ...
+    def __radd__(self, other: torch.Tensor) -> LinearOperator:  # type: ignore[misc]
+        """Operator addition.
+
+        Returns lambda x: lambda x: self(x) + other
```
Comment (ckolbPTB) [inline src/mrpro/operators/LinearOperator.py]: should this be self(x) + other*x (similar in line 176)?

File: src/mrpro/operators/LinearOperator.py [hunk 2]
```diff
@@ -151,21 +152,32 @@ def __matmul__(
         else:
             return OperatorComposition(self, other)
 
-    @overload
-    def __add__(self, other: LinearOperator) -> LinearOperator: ...
+    def __radd__(self, other: torch.Tensor) -> LinearOperator:  # type: ignore[misc]
+        """Operator addition.
+
+        Returns lambda x: lambda x: self(x) + other*x
```
Comment (schuenke) [inline src/mrpro/operators/LinearOperator.py]: one `lambda x:` to much?

**Review by schuenke — APPROVED** [review]
Just did a very very quick review, but lgtm and tests seem to work as expected.

**Review by fzimmermann89 — CHANGES_REQUESTED** [review]
Please hold with merge

**Comment by fzimmermann89** [comment]
This is one way to do it, via a separate (Linear)OperatorTensorSum class.
Alternativly, we could also reuse the IdentityOp:
LinOp + Tensor = LinOpSum(Op, Identity*Tensor), see #430

**Pull request closed without merging**